wealth-management-client/src/app/reports/userviews/userview-grid.component.css

```diff
@@ -0,0 +1,3 @@
+.progress {
```

Comment:
Move this style to styles.css and delete both css files of components.

Reply:
OK.
So will we have only one styles.css for the entire app?

Reply:
Done